packages/admin/blocks-admin/src/blocks/factories/createOneOfBlock.tsx

```diff
@@ -62,13 +62,15 @@ export interface CreateOneOfBlockOptions<T extends boolean> {
     name: string;
     displayName?: React.ReactNode;
     supportedBlocks: Record<BlockType, BlockInterface>;
+    tabLabels?: Record<BlockType, React.ReactNode>;
```

Comment:
I'd name this labels, as it also works for the radio or select variant. And we use it for something different in the future.